### PR TITLE
Brand Colors  [CV2-4616]

### PR DESCRIPTION
**Pull request by brianfleming**
## Description

This PR is another push towards aligning Check with the overall Meedan branding. This is a large number of file changes, but I am trying to limit it to just colors:
- Add new color vars from the design system
- Replace old color vars with new values
- Start to adjust colors around the app to match Meedan branding
- Only non color change: rounding on side navigation to match Meedan.com navigation

References:  [[CV2-4616]](https://meedan.atlassian.net/browse/CV2-4616)

## Type of change

- [x] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

Manually going around the app and looking for anything weird

## Things to pay attention to during code review

This will most likely require future PRs to fix anything that looks odd

### AFTER
Screenshot 2024-05-20 at 12 40 53 PM

## Checklist

- [x] I have performed a self-review of my own code
- [x] I've made sure my branch is runnable and given good testing steps in the PR description
- [x] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [x] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [x] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

[CV2-4616]: https://meedan.atlassian.net/browse/CV2-4616?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by brianfleming** [comment]
Reviewers, this is the color changes I mentioned on the front-end call the other day.
I fully expect this PR to have issues, which is why I would like to get it merged. I think we will only discover weird things through usage. I can then followup with other PRs to fix those things.

As discussed this will start off a uneasy period with the code and Figma being out of sync for a bit, I expect this to not last very long. During this time if you have any difficulties or need color help please reach out.

cc @caiosba for visibility

File: src/app/styles/css/mixins/variables.css
Comment (brianfleming) [inline src/app/styles/css/mixins/variables.css:1]: These color vars should match what you will start to see in Figma

**Review by amoedoamorim — APPROVED** [review]
I have marked 126 "Viewed" checkboxes and did not find any sneaky malicious scripts, just color variables and some styling fixes.

**Comment by brianfleming** [comment]
> I have marked 126 "Viewed" checkboxes and did not find any sneaky malicious scripts, just color variables and some styling fixes.

so my devious plan worked!

thanks for the review @amoedoamorim

**Pull request merged**